moped-editor/src/views/projects/projectView/ProjectPhases.js

```diff
@@ -157,6 +160,13 @@ const ProjectPhases = ({ projectId, data, refetch }) => {
   const [deletePhase, { loading: deleteInProgress }] =
     useMutation(DELETE_PROJECT_PHASE);
 
+  const { data: completionDateData, refetch: refetchCompletionDate } = useQuery(
+    GET_PROJECT_SUBSTANTIAL_COMPLETION_DATE,
+    {
+      variables: { projectId: Number(projectId) },
+    }
+  );
+
```

Comment:
would it be better to include this query in whatever query gets the data that is passed to this component in `ProjectTimeline`? i don't think this is a big deal but i have a slight preference for fewer queries.